src/backend/patches/@mapeo+core+9.0.0-alpha.5.patch

```diff
@@ -21,34 +21,3 @@ index d4cfdfd..41cc2db 100644
 +import quickbit from 'quickbit-universal'
  
  export { quickbit }
-diff --git a/node_modules/@mapeo/core/src/discovery/dns-sd.js b/node_modules/@mapeo/core/src/discovery/dns-sd.js
-index 85d38b0..0f05b79 100644
---- a/node_modules/@mapeo/core/src/discovery/dns-sd.js
-+++ b/node_modules/@mapeo/core/src/discovery/dns-sd.js
-@@ -40,14 +40,21 @@ export class DnsSd extends TypedEmitter {
-       this.#log(`Ignoring service up from self`)
-       return
-     }
--    const address = service.addresses?.find(isIPv4)
--    /* c8 ignore start */
-+
-+    let address = service.addresses?.find(isIPv4)
-+    const { name, port, referer } = service
-+
-+    if (!address) {
-+      if (!referer) {
-+        return
-+      }
-+      address = referer.address
-+    }
-+    
-     if (!address) {
--      this.#log(`service up (${service.name}) with no ipv4 addresses`)
-       return
-     }
--    /* c8 ignore stop */
--    const { name, port } = service
-+
-     this.#log('serviceUp', name.slice(0, 7), address, port)
-     this.emit('up', { port, name, address })
-   }
```

Comment:
No longer necessary because this is removed in `@mapeo/core`. See https://github.com/digidem/mapeo-core-next/pull/550.